src/components/NewGalaxyModal.js

```diff
@@ -0,0 +1,194 @@
+    )
+  }
+
+  render() {
+    const { onDismiss } = this.props
+    const { viewMode } = this.state
+
+    const renderBottomButtons = () => {
+      return h(Fragment, [
+        div({ style: { display: 'flex', margin: '1rem 0 1rem' } }, [
+          div({ style: { flex: 1 } }),
+          this.renderActionButton()
+        ])
+      ])
+    }
+
+    const getMachineDetails = () => {
+      return _.filter(({ name }) => name === 'n1-standard-8', machineTypes)[0]
+    }
+
+    const renderCreateWarning = () => {
+      return h(Fragment, [
+        div({ style: { marginBottom: '1rem' } }, ['Environment consists of an application and cloud compute.']),
+        div({ style: { ...styles.whiteBoxContainer, backgroundColor: colors.accent(.1), boxShadow: Style.standardShadow } }, [
+          div({ style: { display: 'flex', padding: `0.5rem 0 0`, minHeight: 120 } }, [
+            div({ style: { flex: '1', lineHeight: '1.5rem', minWidth: 0, display: 'flex' } }, [
+              span({ style: { marginRight: '0.5rem', marginTop: '0.5rem' } }, [icon('info-circle', { size: 25, color: colors.accent() })]),
+              div([
+                span({ style: styles.headerText }, ['Set up duration']),
+                div({ style: { lineHeight: '24px' } }, [
+                  div(['Creating a cloud environment for Galaxy takes ', span({ style: { fontWeight: 600 } }, ['8-10 minutes.'])]),
+                  div(['You can navigate away, and we will notify you when it\'s ready. '])
+                ]),
+                div({ style: { ...styles.headerText, marginTop: '0.5rem' } }, ['Continuation cost']),
+                div({ style: { lineHeight: '18px' } }, [
+                  div(['Please delete the cloud environment when finished; it will']),
+                  div(['continue to ', span({ style: { fontWeight: 600 } }, ['incur charges ']), 'if it keeps running.'])
+                ])
+              ])
+            ])
+          ])
+        ])
+      ])
+    }
+
+    const renderDeleteWarning = () => {
+      return div({ style: { lineHeight: '22px' } }, [
+        div({ style: { marginTop: '0.5rem' } }, [
+          'Deleting your Cloud Environment will also ',
+          span({ style: { fontWeight: 600 } }, ['delete any files on the associated hard disk']),
+          ' (e.g. results files). Double check that your workflow results were written to ',
+          'the data tab in the workspace before clicking “Delete”'
+        ]),
+        div({ style: { marginTop: '1rem' } }, ['Deleting your Cloud Environment will stop your ',
+          'running Galaxy application and all associated costs. You can create a new Cloud Environment ',
+          'for Galaxy later, which will take 8-10 minutes.'])
+      ])
+    }
+
+    const renderDefaultCase = () => {
+      return h(Fragment, [
+        div(['Environment consists of an application and cloud compute.']),
```

Comment:
UX TOL: Looking over the various screens, I think we might be muddying the idea of 'status' vs 'intention' in a couple places. We should make sure it's clear to the user when we're communicating the current state of the world (status), vs giving the opportunity to change to that state (intention).

For example, it looks like this default view is acting as 'intention' if you don't have an app, but a 'status' if you do. Even if the info isn't wrong (at this point), it's still confusing because it's presented the same.

I know this is just an MVP and will change substantially, but I wanted to bring that up to make sure we have the right frame as we go forward.

Reply:
We did a brief change to have it be a ternary on the word `consists` to either be `will consist` or `consists`.